packages/core/src/auth/sso/ssoAccessTokenProvider.ts

```diff
@@ -188,6 +192,25 @@ export abstract class SsoAccessTokenProvider {
 
             return refreshed
         } catch (err) {
+            const possibleCacheError = DiskCacheError.instanceIf(err)
```

Comment:
This will wrap known filesystem errors during token refresh process and cause them to be handled gracefully eventually by`throwOnRecoverableError()` above